hack/cherry_pick_pull.sh

```diff
@@ -44,8 +44,8 @@ if [[ -z ${GITHUB_USER:-} ]]; then
   exit 1
 fi
 
-if ! which hub > /dev/null; then
-  echo "Can't find 'hub' tool in PATH, please install from https://github.com/github/hub"
+if ! which gh > /dev/null; then
```

Comment:
nit, since we're touching this line:
```suggestion
if ! command -v gh > /dev/null; then
```